### DISCUSSION TITLE
Please tell me how to cover the screen in a window.

### DISCUSSION THREAD

**Discussion by developerDTH** [community/community#96214] [Programming Help]
### Body

I want to create a family link program for PC.
Because I have to stop the computer to study. But I can't physically stop it. I'm so weak-willed that I turn it back on. So I want to make it in software. Even if I'm a beginner, developers are developers, so I want to try making it myself. So I need help. This program should not be able to perform basic functions on your PC. For example, running programs, games, web, etc. And I also need to prevent myself from finding loopholes in this program and terminating it. Please help me.
Currently I can only use Python and JavaScript. However, if you make a perfect program in another language and give it to me, I am willing to learn that language and use and understand it. And please provide a detailed explanation.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
Hey @developerDTH I personally think that is really awesome idea. Doing this actually varies on windows and mac so state what os you are using. Then I guess making this Terminal based application would be actually better as this would not distract you like a application. Also I suggest using python for it
